hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -173,6 +173,70 @@ def hpat_pandas_series_shape_impl(self):
+    """
+    Pandas Series method :meth:`pandas.Series.std` implementation.
+    .. only:: developer
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unboxing
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_str
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unsupported_axis
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unsupported_level
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_std_unsupported_numeric_only
+    Parameters
+    ----------
+    self: :obj:`pandas.Series`
+        input series
+    axis: :obj:`int`, :obj:`str`
+        Axis along which the operation acts
+        0/None - row-wise operation
+        1      - column-wise operation
+        *unsupported*
+    skipna: :obj:`bool`
+        exclude NA/null values
+    level: :obj:`int`, :obj:`str`
+        If the axis is a MultiIndex (hierarchical),
+        count along a particular level, collapsing into a scalar
+        *unsupported*
+    ddof: :obj:`int`
+        Delta Degrees of Freedom.
+        The divisor used in calculations is N - ddof,
+        where N represents the number of elements.
+    numeric_only: :obj:`bool`
+        Include only float, int, boolean columns.
+        If None, will attempt to use everything, then use only numeric data.
+        Not implemented for Series.
+        *unsupported*
+    Returns
+    -------
+    :obj:`scalar`
+         returns :obj:`scalar`
+    """
+    _func_name = 'Method std().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    if not isinstance(self.dtype, types.Number):
+        raise TypingError('{} The object must be a number. Given self.dtype: {}'.format(_func_name, self.dtype))
+
+    if not isinstance(skipna, (types.Omitted, types.Boolean, types.NoneType)) and skipna is not None:
+        raise TypingError('{} The object must be a boolean. Given skipna: {}'.format(_func_name, skipna))
+
+    if not isinstance(ddof, (types.Omitted, int, types.Integer)):
+        raise TypingError('{} The object must be an integer. Given ddof: {}'.format(_func_name, ddof))
+
+    for name, arg in [('axis', axis), ('level', level), ('numeric_only', numeric_only)]:
+        if not isinstance(arg, (types.Omitted, types.NoneType)) and arg is not None:
+            raise TypingError('{} Unsupported parameters. Given {}: {}'.format(_func_name, name, arg))
+
+    def hpat_pandas_series_std_impl(self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None):
+        return math.sqrt(self.var(axis=axis, skipna=skipna, level=level, ddof=ddof, numeric_only=numeric_only))
```

Comment:
Because numpy implementation has `ddof=0` by default, but `ddof=1` is by default for pandas. Implementation of `series.var` takes into account the fact.